packages/components/src/components/Callout/Callout.test.tsx

```diff
@@ -9,12 +9,6 @@ describe("<Callout />", () => {
     expect(screen.getByText("Message here")).toBeInTheDocument();
   });
 
-  it("should render the icon", () => {
-    render(<Callout>Message here</Callout>);
-
-    expect(screen.getByTestId("callout-icon")).toBeInTheDocument();
-  });
```

Comment:
Is this covered by Chromatic, right?

Reply:
Btw, why did you remove the test-id and this test? It is something related to the downgrade?

Reply:
Yeah. The icon is a visual treatment so will be caught there if its missing.

Reply:
> Btw, why did you remove the test-id and this test? It is something related to the downgrade?

We shouldn't be forcing test-ids in our components. That can cause issues with teams using the components and their tests, i.e. they use the same test-id.